docker-compose.yml

```diff
@@ -19,14 +19,14 @@ services:
         fi
     volumes:
       - boot:/mnt/boot
-      - ./config.json:/config.json
+      - ${CONFIG_JSON:-./config.json}:/config.json
   os:
     build:
       context: .
       dockerfile: Dockerfile.template
       args:
-        OS_VERSION: 2.68.1_rev1
-        DEVICE_TYPE: intel-nuc
+        OS_VERSION: ${OS_VERSION:-2.68.1_rev1}
+        DEVICE_TYPE: ${DEVICE_TYPE:-intel-nuc}
     depends_on:
       - provision
     privileged: true
```

Comment:
This commit can be merged with the one that introduces the compose file, there's nothing here to merit a different commit.